src/CommunityToolkit.Maui/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -346,11 +346,13 @@ async Task OnValuePropertyChanged(CancellationToken token)
 
 	void OnValuePropertyNamePropertyChanged()
 	{
+#pragma warning disable IL2026 // NET9.0 - check if this can be fixed
```

Comment:
I have suppressed this for now but we should consider whether we can avoid it

Reply:
I'm fairly confident that we do need to fix this otherwise `ValidationBehavior` won't be safe for Trimming/AOT. 

The MAUI team has said that string-based bindings are no longer supported due to Trimming.

Reply:
I've resolved the builder errors for now by adding the `[RequiresUnreferencedCode]` attribute to `ValidationBehavior`. 

After we release .NET 9 support, we can circle back to this to see if there is a way to avoid using string bindings with `ValidationBehavior`.